docs/production/authentication-methods.md

```diff
@@ -610,14 +610,16 @@ to the root and `engineering` subdomains:
          importing, only the certificate will be displayed (not the private
          key).
 
-### IdP-initiated SAML Logout
+### SAML Logout
 
-Zulip 5.0 introduces beta support for IdP-initiated SAML Logout. The
-implementation has primarily been tested with Keycloak and these
+Zulip 5.0 introduces beta support for both IdP-initiated and SP-initiated
+SAML Logout. The implementation has primarily been tested with Keycloak and these
```

Comment:
This needs editing, since 6.0 introduces SP-initiated. We should probably also remove the "beta" bit at this point; we can just say "Zulip supports both IdP-initiated and SP-initiated SAML Logout."